src/main/java/org/geysermc/discordbot/commands/moderation/ForumPostCommand.java

```diff
@@ -0,0 +1,355 @@
+import org.geysermc.discordbot.util.DicesCoefficient;
+import org.jetbrains.annotations.NotNull;
+
+import java.time.OffsetDateTime;
+import java.util.*;
+
+public class ForumPostCommand extends SlashCommand {
+
+    public ForumPostCommand() {
+        this.name = "post";
+        this.hidden = true;
+        this.help = "Help tool to manage forum posts.";
+        this.guildOnly = true;
+        this.children = new SlashCommand[] {
+                new CreatePostSubCommand(),
+                new RenamePostSubCommand(),
+                new ClosePostSubCommand(),
+                new AddPostTagSubCommand(),
+                new RemovePostTagSubCommand(),
+                new CloseOldPostSubCommand()
+        };
+    }
+
+    @Override
+    protected void execute(SlashCommandEvent event) {
+        // unused
+    }
+
+    public static class CreatePostSubCommand extends SlashCommand {
+        public CreatePostSubCommand() {
+            this.name = "create";
+            this.help = "Create post";
+            this.userPermissions = new Permission[] { Permission.CREATE_PUBLIC_THREADS };
+            this.options = Arrays.asList(
+                    new OptionData(OptionType.STRING, "title", "Add the post title", true),
+                    new OptionData(OptionType.STRING, "issue", "Add the post message/issue", true),
+                    new OptionData(OptionType.USER, "member", "The member to ping in the post", false)
+            );
+        }
+
+        @Override
+        protected void execute(@NotNull SlashCommandEvent event) {
+            Checks.notNull(event.getGuild(), "server");
+
+            String title = event.optString("title", "");
+            OptionMapping memberMapping = event.getOption("member");
+            String targetUser = memberMapping == null ? "" : memberMapping.getAsUser().getAsMention();
+            String issue = event.optString("issue", "");
+
+            if (!targetUser.isEmpty()) {
+                issue += " " + targetUser;
+            }
+
+            ForumChannel forumChannel = ServerSettings.getForumChannel(event.getGuild());
+            if (forumChannel == null) {
+                event.reply("Forum channel not found.").queue();
+                return;
+            }
+
+            forumChannel.createForumPost(title, MessageCreateData.fromContent(issue))
+                .queue(
+                    unused -> event.reply("Post is created").queue(),
+                    error -> event.reply("Could not create post").queue()
+                );
+        }
+    }
+
+    public static class RenamePostSubCommand extends SlashCommand {
+        public RenamePostSubCommand() {
+            this.name = "rename";
+            this.help = "Rename post";
+            this.userPermissions = new Permission[] { Permission.CREATE_PUBLIC_THREADS };
+            this.options = Collections.singletonList(new OptionData(OptionType.STRING, "title", "change the forum title", true));
```

Comment:
```suggestion
            this.options = Collections.singletonList(new OptionData(OptionType.STRING, "title", "change the forum title", true).setMaximumLength(Channel.MAX_NAME_LENGTH));
```

This ensures it's 100 or less characters, requires an import though.